README.md

```diff
@@ -83,14 +83,18 @@ def test_targets_version(
     # Initialize our repository: modify targets version and make sure the version is included in snapshot
     init_data, repo = server.new_test(client.test_name)
     repo.targets.version = 7
-    repo.update_snapshot()  # snapshot v2
+    repo.publish([Targets.type, Snapshot.type, Timestamp.type])
```

Comment:
this example will fail in `publish()` as the URL version (2) and metadata version (7) do not match. Same thing happens later when v7 is published.

You could change this example to actually test that (url version and metadata version mismatch) as it's simpler than the current correct example would be. Alternatively you must publish the metadata versions at correct URLs.

Reply:
Updated in https://github.com/theupdateframework/tuf-conformance/pull/188/commits/df9b94aae732c4ebbadb18c42a4a6032cefdf646.